skt/mls.py

```diff
@@ -175,11 +180,112 @@ def get_feature_list(model, model_library: str):
             )
 
         process_model_meta = subprocess.Popen(
-            shlex.split(cmd_load_meta_to_s3), stdout=subprocess.PIPE, stdin=subprocess.PIPE
+            shlex.split(cmd_load_meta_to_s3), stdout=subprocess.PIPE, stdin=subprocess.PIPE,
         )
         process_model_meta.wait()
         if process_model_meta.returncode != 0:
             raise MLSModelError(f"Load model_meta(meta.json) to S3 ({s3_path}) is FAILED")
 
     finally:
         shutil.rmtree(model_path, ignore_errors=True)
+
+
+def get_meta_table(meta_table: str, aws_env: AWSENV = AWSENV.STG.value) -> Dict[str, Any]:
+    """
+    Get a meta_table information
+    Args. :
+        - meta_table   :   (str) the name of meta_table
+        - aws_env      :   (str) AWS ENV in 'stg / prd / dev' (default is 'stg') (default : 'stg)
+    Returns :
+        - Dictionary value of meta_table (id / name / description / schema / items / created_at / updated_at)
+    """
+    assert type(meta_table) == str
+    assert type(aws_env) == str
+
+    url = f"https://ab-internal.{aws_env}.sktmls.com{MLS_META_API_URL}/{meta_table}"
+
+    if requests.get(url).json().get("results"):
+        return requests.get(url).json()["results"]
+    else:
+        raise MLSModelError(requests.get(url).json()["error"])
```

Comment:
API 호출을 여러번 하는 코드네요.

```suggestion
    response = requests.get(url).json()
    results = response.get("results")
    if not results:
        raise MLSModelError(response.get("error"))
    return results
```